### DISCUSSION TITLE
Cannot using more than one TIMESTAMP in same table

### DISCUSSION THREAD

**Discussion by ardinusawan** [PyMySQL/mysqlclient#641] [Trash]
Getting error 
`_mysql_exceptions.OperationalError: (1067, "Invalid default value for 'date_updated'")`

**Comment by methane** [comment]
google may help you

**Comment by MathieuGrosso** [comment]
same error, thanks for nothing

**Comment by methane** [comment]
thanks for spam😠
driver maintainer is not free tech support for MySQL.
